src/node/validation_cache_args.cpp

```diff
@@ -18,9 +21,10 @@ namespace node {
 void ApplyArgsManOptions(const ArgsManager& argsman, ValidationCacheSizes& cache_sizes)
 {
     if (auto max_size = argsman.GetIntArg("-maxsigcachesize")) {
+        size_t clamped_size_each = std::max<int64_t>(*max_size, 0) / 2 * (1 << 20);
```

Comment:
In commit "validationcaches: Use size_t for sizes" (8e407571e99b0d4aad7f225b8197f35c6b6cea45)

Deleted comment about if size "is set to zero, setup_bytes creates the minimum possible cache (2 elements)", may be useful to keep to explain why 0 is the floor